### PR TITLE
"keyword" parameter for full-text search

### PR DESCRIPTION
**Pull request by tdddblog**
"keyword" parameter for full-text search

### CODE REVIEW

File: swagger.yml [hunk 1]
```diff
@@ -8,7 +8,7 @@ info:
   description: |
     Federated PDS API which provides actionable end points standardized
     between the different nodes.
-  version: 0.3.dev
+  version: keywords
```
Comment (jordanpadams) [inline swagger.yml:11]: @tloubrieu-jpl @tdddblog shouldn't this come from swaggerhub?

Comment (tdddblog) [reply swagger.yml:11]: As discussed with @tloubrieu-jpl, I used "SYNC" button in SwggerHub to create this branch (SwggerHub created it). I'm not sure how `swagger.yml` is usually merged.

**Comment by jordanpadams** [comment]
@tdddblog to confirm, we can close this PR since we discussed how to do this through SwaggerHub?

**Comment by tdddblog** [comment]
@jordanpadams Yes, if my changes are in `swagger.yml` and if codegen (mvn install) ran against the new `swagger.yml`. Otherwise `registry-api-service` will not compile.

**Pull request closed without merging**